lib/core_ext/string/capitalize_first.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class String
+  # Capitalizes the first letter of the string.
+  #
+  # Inspired by https://github.com/rails/rails/blob/6f0d1ad14b92b9f5906e44740fce8b4f1c7075dc/activesupport/lib/active_support/inflector/methods.rb#L166
+  #
+  # @return [String] a string with the first letter capitalized
+  def capitalize_first
+    return "" if empty?
+
+    self[0].upcase + self[1..]
```

Comment:
can we just use [String#capitalize](https://ruby-doc.org/3.2.2/String.html#method-i-capitalize)?

Reply:
ah no, there's a difference. Ok.